.github/workflows/make-rpm.yml

```diff
@@ -24,3 +24,10 @@ jobs:
           go-version: ${{ matrix.go }}
       - name: Build rpm
         run: CONTAINER_RUNTIME=docker make test-rpmbuild
+      - name: Upload linux binary and rpms
+        uses: actions/upload-artifact@v3
+        with:
+          name: linux binary and rpm (${{ matrix.os }}
```

Comment:
this is not exactly `matrix.os` as this was built inside a container image (forgot if the image is centos, ubi, ..)